docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,877 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Datasets - Reduced data, IRFs, models \n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "`gammapy.datasets` are a crucial part of the gammapy API. `datasets` constitute `DL4` data - binned counts, IRFs, models and the associated likelihoods. `Datasets` from the end product of the `makers` stage, see [makers notebook](makers.ipynb), and are passed on to the `Fit` or estimator classes for modelling and fitting purposes.\n",
+    "\n",
+    "There are three types of `Dataset` available\n",
+    "- `MapDataset`: Binned counts and IRFs on a `WCSGeom` spatial geom and an energy axis; used for 3D analysis, supports cash likelihood\n",
+    "- `SpectrumDataset`: Binned counts and IRFs on a `RegionGeom` and energy axis; used for 1D spectral analysis; supports cash likelihood\n",
+    "- `FluxPointDataset`: Directly supports fitting of pre-computed flux points, no IRF convolution is performed during fitting, and supports `chi2` statistics\n",
+    "\n",
+    "`MapDataset` and `SpectrumDataset` in turn support `MapDatasetOnOff` and `SpectrumDatasetOnOff`, respectively, which use `wstat` statistics and should be used when the background is estimated from real off counts. These store, additionally, the `counts_off`, `acceptance` and `acceptance_off data`.\n",
+    "\n",
+    "\n",
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-08-31T15:44:56.843097Z",
+     "iopub.status.busy": "2021-08-31T15:44:56.840387Z",
+     "iopub.status.idle": "2021-08-31T15:44:58.004359Z",
+     "shell.execute_reply": "2021-08-31T15:44:58.003875Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import astropy.units as u\n",
+    "from regions import CircleSkyRegion\n",
+    "from astropy.coordinates import SkyCoord\n",
+    "from gammapy.datasets import (\n",
+    "    MapDataset,\n",
+    "    SpectrumDataset,\n",
+    "    Datasets,\n",
+    "    FluxPointsDataset,\n",
+    ")\n",
+    "from gammapy.data import DataStore\n",
+    "from gammapy.maps import WcsGeom, RegionGeom, MapAxes, MapAxis, Map\n",
+    "from gammapy.modeling.models import SkyModel, PowerLawSpectralModel\n",
+    "from gammapy.estimators import FluxPoints"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## MapDataset\n",
+    "\n",
+    "The counts, exposure, background, masks, and IRF maps are bundled together in a data structure named `MapDataset`. While the `counts`, and `background` maps are binned in reconstructed energy must have the same geometry, the IRF maps can have a different spatial (coarsely binned and larger) geometry and spectral range (binned in true energies). It is usually recommened that the true energy bin should be larger and more finely sampled and the reco energy bin.\n",
```

Comment:
something is missing in this sentence:
"While the `counts`, and `background` maps are binned in reconstructed energy must [...]"

Reply:
Also "recommened' -> "recommended" ;)